flink-ml-python/pyflink/ml/lib/feature/binarizer.py

```diff
@@ -0,0 +1,68 @@
+################################################################################
+#  Licensed to the Apache Software Foundation (ASF) under one
+#  or more contributor license agreements.  See the NOTICE file
+#  distributed with this work for additional information
+#  regarding copyright ownership.  The ASF licenses this file
+#  to you under the Apache License, Version 2.0 (the
+#  "License"); you may not use this file except in compliance
+#  with the License.  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+# limitations under the License.
+################################################################################
+from typing import Tuple
+
+from pyflink.ml.core.param import ParamValidators, Param, FloatArrayParam
+from pyflink.ml.core.wrapper import JavaWithParams
+from pyflink.ml.lib.feature.common import JavaFeatureTransformer
+from pyflink.ml.lib.param import HasInputCols, HasOutputCols
+
+
+class _BinarizerParams(
+    JavaWithParams,
+    HasInputCols,
+    HasOutputCols
+):
+    """
+    Params for :class:`Binarizer`.
+    """
+
+    THRESHOLDS: Param[Tuple[float, ...]] = FloatArrayParam(
+        "thresholds",
+        "Binarization thresholds, when the input continuous feature value"
+        " is greater than the threshold, it will be set to 1.0, else 0.0.",
```

Comment:
Let's keep the description the same as that in Java. 

Besides, could you please check whether the other comments in the previous review about the Java implementation have corresponding components in Python? Let's also modify those components to keep them aligned with the implementation in Java.